<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWindow</class>
 <widget class="QMainWindow" name="ControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Painel de Controle</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #2b2b2b;
}
QGroupBox {
	background-color: #34495e;
	border: 2px solid #1e3a5f;
	border-radius: 5px;
	margin-top: 10px;
	font-weight: bold;
	color: white;
}
QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top center;
	padding: 0 10px;
	color: white;
}
QLabel {
	color: #ecf0f1;
}
QLineEdit {
	background-color: #3d4a5c;
	border: 1px solid #1e3a5f;
	border-radius: 4px;
	padding: 4px;
	color: white;
}
QLineEdit:focus {
	border: 2px solid #4a90e2;
	background-color: #455a6f;
}
QSlider::groove:horizontal {
	background-color: #3d4a5c;
	height: 6px;
	border-radius: 3px;
}
QSlider::handle:horizontal {
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #4a90e2, stop:1 #357abd);
	border: 2px solid #1e3a5f;
	width: 18px;
	height: 18px;
	margin: -6px 0;
	border-radius: 9px;
}
QSlider::handle:horizontal:hover {
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #5ba0f2, stop:1 #4580cd);
}
QSlider::sub-page:horizontal {
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #4a90e2, stop:1 #357abd);
	border-radius: 3px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <spacer name="verticalSpacer_top">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_main">
      <item>
       <spacer name="horizontalSpacer_left">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>280</height>
         </size>
        </property>
        <property name="title">
         <string>Painel de Controle do Trem</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="topMargin">
          <number>30</number>
         </property>
         <property name="rightMargin">
          <number>30</number>
         </property>
         <property name="bottomMargin">
          <number>30</number>
         </property>
         <item>
          <widget class="QLabel" name="label_info">
           <property name="styleSheet">
            <string notr="true">color: #bdc3c7; font-style: italic; font-size: 11px;</string>
           </property>
           <property name="text">
            <string>* Deixe o ID vazio ou 0 para manipular todos os trens</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layoutId">
           <item>
            <widget class="QLabel" name="label_id">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>ID do Trem:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineTrainId">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="placeholderText">
              <string>Ex: 1</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layoutSpeed">
           <item>
            <widget class="QLabel" name="label_speed">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Velocidade:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="sliderSpeed">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelSpeedValue">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #ecf0f1;</string>
             </property>
             <property name="text">
              <string>1</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_middle">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layoutButtons">
           <property name="spacing">
            <number>15</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnTrainOff">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #e74c3c, stop:1 #c0392b);
	color: white;
	border-radius: 4px;
	border: 2px solid #8b0000;
}
QPushButton:hover {
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #f55c4c, stop:1 #d14939);
	border: 2px solid #a00000;
}
QPushButton:pressed {
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #c0392b, stop:1 #a93226);
}</string>
             </property>
             <property name="text">
              <string>PARAR</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnTrainOn">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #2ecc71, stop:1 #27ae60);
	color: white;
	border-radius: 4px;
	border: 2px solid #1e8449;
}
QPushButton:hover {
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #3edc81, stop:1 #37be70);
	border: 2px solid #229954;
}
QPushButton:pressed {
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #27ae60, stop:1 #219150);
}</string>
             </property>
             <property name="text">
              <string>CONTINUAR</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_right">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_bottom">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #34495e; color: white;</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: #34495e; color: white;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderSpeed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelSpeedValue</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>

